src/explore-education-statistics-frontend/src/components/Link.tsx

```diff
@@ -20,20 +25,40 @@ const Link = ({
   prefetch,
   to,
   href,
+  analytics,
   unvisited = false,
   ...props
 }: LinkProps) => {
+  const handleAnalytics = () => {
+    if (analytics) {
+      logEvent(
+        analytics.category || window.location.pathname,
+        analytics.action,
+      );
+    }
+  };
+
   return (
     <RouterLink href={href || to} as={as} prefetch={prefetch}>
       <a
         {...props}
+        role="link"
+        tabIndex={-1}
```

Comment:
Shouldn't need to add either `role` or `tabIndex` props as this is an anchor element already. In fact, a `tabIndex` of -1 is definitely an accessibility no-no as it means you can't press Tab to focus the link.

If the linter is telling you need to add these props, you should disable the lints

Reply:
Yeah, it was the linter complaining. I'll remove them.